src/operators.jl

```diff
@@ -576,6 +582,16 @@ function Base.:-(x::AbstractArray{T}) where {T <: _JuMPTypes}
     return ret
 end
 
+# Fix https://github.com/JuliaLang/julia/issues/32374 as done in
+# https://github.com/JuliaLang/julia/pull/32375. This hack should
+# be removed once we drop Julia v1.0.
+function Base.:-(A::Symmetric{<:JuMP.AbstractVariableRef})
```

Comment:
It would be nice to have a comment on https://github.com/JuliaLang/julia/pull/32375 that this is the right fix, or even better have the PR merged before we patch it into JuMP.

Reply:
Even if it is merged, it will be long before we can assume that the fix is in people binaries. I agree that we could wait for a review from the Julia PR.

Reply:
It is merged

Reply:
It is merged

Reply:
LGTM. Please fix the merge conflicts.